### ISSUE TITLE
Train on Images Only

### ISSUE THREAD

**Issue by RJVisee44** [davheld/GOTURN#82]
Is there a way to train on images only? Eg. train on ImageNet only.

**Comment by hjl240** [comment]
yes, find src/train/train.cpp, and comments the lines relevant to video data loader and video trainer.
